app/models/document_capture_session.rb

```diff
@@ -16,6 +16,10 @@ def store_result_from_response(doc_auth_response)
     save!
   end
 
+  def expired?
+    requested_at + Figaro.env.doc_capture_request_valid_for_minutes.to_i.minutes < Time.zone.now
```

Comment:
This will error if `requested_at` is nil, can we add a guard or something here?

Reply:
> This will error if `requested_at` is nil, can we add a guard or something here?

> Also this seems like a good candidate for a unit test, can we get one of those?

Guard and associated test added in rebased b5f1334ea68533771810bbf9b83022da279d3482.